common/services/app/google-analytics.tsx

```diff
@@ -11,13 +11,14 @@ type Props = {
   data: {
     toggles?: Toggles;
   };
+  hasAnalyticsConsent: boolean;
 };
 
 // We send toggles as an event parameter to GA4 so we can determine the condition in which a particular event took place.
 // GA4 now limits event parameter values to 100 characters: https://support.google.com/analytics/answer/9267744?hl=en
 // So instead of sending the whole toggles JSON blob, we only look at the "test" typed toggles and send a concatenated string made of the toggles' name
 // , preceeded with a! if its value is false.
-function createToggleString(toggles: Toggles | undefined): string | null {
+function createABToggleString(toggles: Toggles | undefined): string | null {
```

Comment:
From what I gathered, this only returns AB test strings, so I renamed to clarify.